[MPNet P0] add PretrainedConfig and unit test

robinbg opened the pull request:
<!-- Demo: https://github.com/PaddlePaddle/PaddleNLP/pull/26 -->
### PR types
<!-- One of [ New features | Bug fixes | Function optimization | Performance optimization | Breaking changes | Others ] -->

### PR changes
<!-- One of [ Models | APIs | Docs | Others ] -->

### Description
<!-- Describe what this PR does -->

robinbg committed:
Create configuration.py

robinbg committed:
Merge branch 'PaddlePaddle:develop' into develop

robinbg committed:
Create configuration.py

robinbg committed:
Update modeling.py

robinbg committed:
Update __init__.py

robinbg committed:
Merge branch 'PaddlePaddle:develop' into robinbg-patch-6

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/PaddlePaddle/PaddleNLP?pullRequest=4941) <br/>All committers have signed the CLA.

sijunhe reviewed (commented):
您好，欢迎参加飞桨黑客松。这个PR是一个好的开始，后续需要写好test_modeling和test_tokenizer单测，并且做到单测通过

robinbg committed:
Update modeling.py

robinbg committed:
Update configuration.py

robinbg committed:
Update modeling.py

robinbg committed:
Update modeling.py

robinbg committed:
Update modeling.py

robinbg committed:
Update modeling.py

robinbg commented:
您好，请问可以帮忙看看为什么copyright过不了吗？我本地的pre-commit是可以过的

robinbg committed:
Update modeling.py

robinbg committed:
Update configuration.py

robinbg committed:
Update modeling.py

robinbg committed:
Update configuration.py

robinbg committed:
Update modeling.py

robinbg committed:
Update tokenizer.py

sijunhe commented on paddlenlp/transformers/megatronbert/configuration.py:
```diff
@@ -0,0 +1 @@
+pass
```
与PR无关，请删除

sijunhe reviewed (commented):
您好。PR需要添加test_modeling和test_tokenizer的单测

sijunhe commented:
> 您好，请问可以帮忙看看为什么copyright过不了吗？我本地的pre-commit是可以过的

lint显示tokenizer.py文件的Import 顺序有一些问题，修改一下即可

![image](https://user-images.githubusercontent.com/11987277/221089262-1581bb5c-f21d-446c-a481-5b23542d2951.png)

robinbg committed:
Update tokenizer.py

robinbg committed:
Delete configuration.py

robinbg committed:
Update modeling.py

robinbg committed:
On branch robinbg-patch-6
 Your branch is up to date with 'origin/robinbg-patch-6'.
 Changes to be committed:
	new file:   tests/transformers/mpnet/__init__.py
	new file:   tests/transformers/mpnet/test_modeling.py
	new file:   tests/transformers/mpnet/test_tokenizer.py

robinbg committed:
On branch robinbg-patch-6
 Your branch is up to date with 'origin/robinbg-patch-6'.
 Changes to be committed:
	modified:   paddlenlp/transformers/mpnet/modeling.py

robinbg committed:
On branch robinbg-patch-6
 Your branch is up to date with 'origin/robinbg-patch-6'.
 Changes to be committed:
	modified:   tests/transformers/mpnet/test_tokenizer.py
